frontend/HelperFuncs/utils.js

```diff
@@ -15,18 +14,86 @@ export const submitProfile = async (userData) => {
             },
             body: JSON.stringify(userData)
         });
-    } catch(error) {
-        console.log('Error adding profile: ', error)
+        return response;
+    } 
+    catch(error) {
+        console.error('Error adding profile: ', error)
     }
 }
 
 
+/* 
+helper function to make put request to update user data with profile info 
+from account creation form
+*/
+export const populateAccount = async (userData, userId) => { 
+    try {
+        const response = await fetch(`${import.meta.env.VITE_BACKEND_ADDRESS}/profiles/${userId}`, {
```

Comment:
unused variable. Make this
- `const _ = await ....`, or 
- `await ....`

Reply:
good point, I will include that as a change in my next pr